bokeh/io/export.py

```diff
@@ -232,7 +232,7 @@ def get_svgs(obj, driver=None, timeout=5, **kwargs):
     with _tmp_html() as tmp:
         html = get_layout_html(obj, **kwargs)
         with io.open(tmp.path, mode="wb") as file:
-            file.write(html.encode('latin-1'))
+            file.write(html.encode('utf-8'))
```

Comment:
I do want to still check on this a bit more. Setting to `latin-1` is atypical so we should make sure there was not some reason for it.

Reply:
The changes from `latin-1` to 'utf-8' is not a light hearth one, for me. Usually there is a valid reason for this.

Reply:
Possibly :) It might also just be some historical accident. I can't find any documentation for the choice, unfortunately. All the current tests pass either way, I would just want to experiment locally with SVG export some in addition.